camille/inputs/bazefetcher.py

```diff
@@ -0,0 +1,41 @@
+from pandas import DataFrame, Timestamp, read_json
+from os import path, walk, listdir
+import dateutil
+
+def _bazefetcher_time_filter(config, filename):
+    filename = path.split(filename)[-1]
+    filename = path.splitext(filename)[0]
+
+    try:
+        s, e = filename.split('_')[-2:]
+    except ValueError:
+        return False
+
+    start = Timestamp(dateutil.parser.parse(s.replace('.',':')))
+    end = Timestamp(dateutil.parser.parse(e.replace('.',':')))
+
+    return config['start'] <= start and end < config['end']
```

Comment:
This rubs me the wrong way somehow. For me it makes more sense if this method takes in start_date and end_date instead of config, so it doesn't have to know about the config-structure.